external/fv3fit/tests/keras/test_pure_keras.py

```diff
@@ -0,0 +1,45 @@
+from fv3fit.keras._models.shared.pure_keras import PureKerasModel
```

Comment:
would be nice to delete these tests...since they are pretty involved and similar to https://github.com/ai2cm/fv3net/blob/master/external/fv3fit/tests/keras/test_adapters.py#L48.